Zip_Dockerfile.twig

```diff
@@ -0,0 +1,65 @@
+{% set shopwareVersion = imageName|split('_')[1] %}
+{% set shopwareUrl = imageName|split('_').slice(2, 5).join('_') %}
+{% set mysqlVersion = imageName|split('_')[5] %}
+{% set phpVersion = imageName|split('_')[6] %}
+{% set consentManagerVersion = imageName|split('_')[7] %}
+{% set consentManagerRequired = consentManagerVersion != 'none' %}
+
+# Check out shopware source
+FROM alpine/git:latest AS clone
+
+RUN wget --header="User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_0) AppleWebKit/600.1.17 (KHTML, like Gecko)a Version/8.0 Safari/600.1.17" https://releases.shopware.com/{{ shopwareUrl }}.zip -O install.zip && unzip install.zip -d /shopware
+RUN cd /shopware
+
+{% if consentManagerRequired %}
+    RUN cd /shopware/custom/plugins && \
+    git clone https://github.com/shopware5/SwagCookieConsentManager.git SwagCookieConsentManager --depth=1 --branch={{ consentManagerVersion }}
+{% endif %}
+
+
+# Inherit files, install NodeJS-dependencies
+FROM node:10 AS js-deps
+
+COPY --from="clone" /shopware /shopware
+
+WORKDIR /shopware
+
+RUN npm install --prefix ./themes && \
+    npm install --prefix ./themes/Frontend/Responsive && \
+    touch .make.install.npm-dependencies
+
+
+FROM ghcr.io/shopware5/docker-images-testing/running:shopware_{{ shopwareVersion }}_{{ mysqlVersion }}_{{ phpVersion }}
+
+COPY --from="js-deps" /shopware /shopware
+
+WORKDIR /shopware
+
+ENV COMPOSER_ALLOW_SUPERUSER=1
+ENV DB_USER=root
+ENV DB_PORT=3306
+ENV DB_NAME=shopware
+ENV DB_HOST=127.0.0.1
+ENV DB_PASSWORD=root
+ENV SW_HOST=localhost
+ENV SW_BASE_PATH=
+
+RUN /bin/bash -c "/usr/bin/mysqld_safe  &" \
+    && sleep 20 \
```

Comment:
hardcoded sleep
If possible please replace with a reliable solution
if that is not possible, then thats fine as well

Reply:
Maybe it would be possible to use something like this: https://github.com/FriendsOfShopware/platform-plugin-dev-docker/blob/main/rootfs/opt/bin/start-mysql